### ISSUE TITLE
[HW3] 추가적인 패키지 설치가 가능할까요?

### ISSUE THREAD

**Issue by jaeyoungleeee** [swsnu/swppfall2022#101]
안녕하세요! 

이번 과제 때 사용할 수 있는 라이브러리의 범위가 어디까지인지 궁금합니다.
create-react-app 에서 제공되는 것까지인 걸까요? 혹은 추가적인 패키지 설치가 가능할지 궁금합니다.

감사합니다!!

**Comment by JadenJSPark** [comment]
저는 `package.json`에 잘 추가되어 `yarn install` 이후 문제없이 동작한다면 추가적인 패키지 설치가 가능한 것으로 이해했습니다.

**Comment by ccdc-git** [comment]
네. npm이나 yarn을 이용해서 추가적인 패키지를 사용하셔도 됩니다. 다만 yarn과 node버전에 따라 오류가 날수 있으니 사용하지 않는것을 권장하고 docker 환경에서 `yarn`이나 `yarn install`을 통해 패키지를 다운로드 받고 제대로 실행되는지 확인해주세요.
